sympy/solvers/tests/test_recurr.py

```diff
@@ -184,6 +184,10 @@ def test_rsolve():
             (C0*((sqrt(-a**2 + 1) - 1)/a)**n +
              C1*((-sqrt(-a**2 + 1) - 1)/a)**n)).simplify() == 0
 
+    assert rsolve((k + 1)*y(k), y(k)) is None
+    assert (rsolve((k + 1)*y(k) + (k + 3)*y(k + 1) + (k + 5)*y(k + 2), y(k))
+            is None)
```

Comment:
So, I guess here also the test says we are not able to find the solution, so we are returning `None`.
I would rather `XFAIL` these in a separate function, so that it may eventually pass someday, when `recurr.py` module is improved. Is there anything stopping us from doing this?